inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/result.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "shared_test_classes/single_layer/result.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace InferenceEngine;
+
+namespace {
+TEST_P(ResultLayerTest, CompareWithRefs) {
+    Run();
+}
```

Comment:
Please move this block to header file in inference-engine/tests/functional/plugin/shared/include/single_layer_tests